src/python/pants/backend/jvm/subsystems/zinc.py

```diff
@@ -244,42 +246,42 @@ def underlying_dist(self):
     return self._zinc_factory.dist
 
   @memoized_property
-  def compiler_bridge(self):
-    """Return the path to the Zinc compiler-bridge jar.
+  def _compiler_bridge(self):
+    """Return a ClasspathEntry for the Zinc compiler-bridge jar.
 
-    :rtype: str
+    :rtype: ClasspathEntry
     """
     return self._zinc_factory._compiler_bridge(self._products)
 
   @memoized_property
-  def compiler_interface(self):
-    """Return the path to the Zinc compiler-interface jar.
+  def _compiler_interface(self):
+    """Return a ClasspathEntry for the Zinc compiler-interface jar.
 
-    :rtype: str
+    :rtype: ClasspathEntry
     """
     return self._zinc_factory._compiler_interface(self._products)
 
   @memoized_property
   def scala_compiler(self):
-    """Return the path to the scala compiler jar.
+    """Return a ClasspathEntry for the scala compiler jar.
 
-    :rtype: str
+    :rtype: ClasspathEntry
     """
     return self._zinc_factory._scala_compiler(self._products)
 
   @memoized_property
   def scala_library(self):
-    """Return the path to the scala library jar (runtime).
+    """Return a ClasspathEntry for the scala library jar (runtime).
 
-    :rtype: str
+    :rtype: ClasspathEntry
```

Comment:
I don't know how public we view this API - I'm fine with this change, but others may not be?

Reply:
It's not marked `:API: public`, so should be good.